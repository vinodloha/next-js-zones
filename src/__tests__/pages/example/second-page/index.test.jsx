import React from 'react'
import renderer from 'react-test-renderer'
import App from '../../../../pages/example/second-page'

describe('Pages:SecondPage', () => {
  it('renders a second example page', () => {
    const component = renderer.create(<App />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
