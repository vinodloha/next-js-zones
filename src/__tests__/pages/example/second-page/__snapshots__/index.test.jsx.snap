// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages:SecondPage renders a second example page 1`] = `
<div
  className="jsx-3583494339 text-center"
>
  <main
    className="jsx-3583494339"
  >
    <p
      className="jsx-3583494339"
    >
      This is another page at a different URL.
    </p>
    <p
      className="jsx-3583494339"
    >
      Return to the
      <a
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        homepage
      </a>
    </p>
  </main>
</div>
`;
